snap-server.cabal

```diff
@@ -91,7 +91,7 @@ Library
     bytestring                          >= 0.9.1    && < 0.11,
     bytestring-builder                  >= 0.10.4   && < 0.11,
     case-insensitive                    >= 1.1      && < 1.3,
-    clock                               >= 0.4.1    && < 0.6,
+    clock                               >= 0.4.1    && < 0.7,
```

Comment:
clock >= 0.6 breaks snap on ubuntu. We are waiting to hear back from clock's author on this: https://github.com/corsis/clock/issues/36